src/m365/context/commands/context-init.ts

```diff
@@ -0,0 +1,66 @@
+
+import * as fs from 'fs';
+import { Logger } from '../../../cli/Logger';
+import Command from '../../../Command';
+import { M365RcJson } from '../../base/M365RcJson';
+import commands from '../commands';
+
+class ContextInitCommand extends Command {
```

Comment:
Let's use AnonymousCommand as the base class since we don't need to be authenticated to use this command